Source/Core/Common/AndroidAnalytics.h

```diff
@@ -0,0 +1,26 @@
+// Copyright 2016 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+#pragma once
+
+#include <string>
+
+#include "Common/Analytics.h"
+
+namespace Common
+{
+typedef void (*SendReport)(std::string, std::string);
+void AndroidSetReportHandler(SendReport function);
+
+class AndroidAnalyticsBackend : public AnalyticsReportingBackend
+{
+public:
+  AndroidAnalyticsBackend(const std::string& endpoint);
+
+  ~AndroidAnalyticsBackend() override;
+  void Send(std::string report) override;
+
+protected:
+  std::string endpoint;
```

Comment:
This should probably be `m_endpoint`, and it doesn't look like it needs to be `protected` (`private` should be enough; although, it seems you just copied this from one of the other backends which also make it `protected`).